pyproject.toml

```diff
@@ -59,6 +59,9 @@ anthropic = [
 gemini = [
     'google-generativeai'
 ]
+local-llm = [
+    'vllm >= 0.5.0'
```

Comment:
I think lower versions should also work, but since it's a rapidly developing library, we should use the newer version.

Reply:
We generally try to keep these as loose as possible, so that if for whatever reason a user needs to use an old version of some library, they'll still be able to use langcheck. So if the older versions also work, I think we can just make this
```
local-llm = [
    'vllm >= 0.5.0'
]
```
(In most cases pip will install the newest version anyway)